assets/js/src/segments/dynamic/dynamic_segments_filters/subscriber_score.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useEffect } from 'react';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+import MailPoet from 'mailpoet';
+import Select from 'common/form/select/select';
+import { Grid } from 'common/grid';
+import Input from 'common/form/input/input';
+
+import {
+  WordpressRoleFormItem,
+} from '../types';
+
+export enum SubscriberScoreOperator {
+  HIGHER_THAN = 'higherThan',
+  LOWER_THAN = 'lowerThan',
+  EQUALS = 'equals',
+  NOT_EQUALS = 'not_equals',
+  UNKNOWN = 'unknown',
+  NOT_UNKNOWN = 'not_unknown',
+}
+
+const availableOperators = [
+  SubscriberScoreOperator.HIGHER_THAN,
+  SubscriberScoreOperator.LOWER_THAN,
+  SubscriberScoreOperator.EQUALS,
+  SubscriberScoreOperator.NOT_EQUALS,
+  SubscriberScoreOperator.UNKNOWN,
+  SubscriberScoreOperator.NOT_UNKNOWN,
+];
+
+type Props = {
+  filterIndex: number;
+}
+
+export function validateSubscriberScore(formItems: WordpressRoleFormItem): boolean {
+  if (
+    formItems.operator === SubscriberScoreOperator.HIGHER_THAN
+    || formItems.operator === SubscriberScoreOperator.LOWER_THAN
+    || formItems.operator === SubscriberScoreOperator.EQUALS
+    || formItems.operator === SubscriberScoreOperator.NOT_EQUALS
+  ) {
+    const numericValue = Number(formItems.value);
+    const numericFloatValue = Number(parseFloat(formItems.value));
```

Comment:
You don't need to use `Number` here, because `parseFloat` already returns a number. https://d.pr/i/UOfSlV but you don't need to update the pull request now.